src/Stratis.Bitcoin/Configuration/Settings/ConsensusSettings.cs

```diff
@@ -40,6 +46,7 @@ public ConsensusSettings(NodeSettings nodeSettings)
             this.UseCheckpoints = config.GetOrDefault<bool>("checkpoints", true, this.logger);
             this.BlockAssumedValid = config.GetOrDefault<uint256>("assumevalid", nodeSettings.Network.Consensus.DefaultAssumeValid, this.logger);
             this.MaxTipAge = config.GetOrDefault("maxtipage", nodeSettings.Network.MaxTipAge, this.logger);
+            this.MaxBlockMemoryInMB = config.GetOrDefault("maxblkmem", 200, this.logger);
```

Comment:
I'd probably add min value and throw here if it's too low. 

Min value should be smth like: `max block size * amount of blocks we expect to be generated in 1h`

Reply:
@noescape00 good idea! How about 50-60 blocks then? We dont actually need to add min property just check that `maxblkmem` isn't less than 60MB (set as a `const`)?

Reply:
We can't hardcode the amount of blocks since it will differ between the chains, better to calculate it based on the target spacing

Reply:
Lets just put the min value to 200 easy and will avoid complications, for now I am happy for this as it is.

Reply:
@mikedennis submit a second PR that throws if size is less then 200.

Reply:
It should limit the amount of blocks you download at any given moment.